git-cms-addpkg

```diff
@@ -9,21 +9,59 @@ case `uname` in
 esac
 
 usage () {
-  $ECHO "git cms-addpkg [options] Subsystem/Package [Subsystem/Package ...] "
-  $ECHO "git cms-addpkg [options] -f FILE"
+  COMMAND_NAME=$1
+  $ECHO "git $COMMAND_NAME [options] Subsystem/Package [Subsystem/Package ...] "
+  $ECHO "git $COMMAND_NAME [options] -f FILE"
   $ECHO
   $ECHO "Options:"
   $ECHO "-h                 \tthis help message"
   $ECHO
   $ECHO "-d, --debug        \tenable debug output"
   $ECHO "-f, --file FILE    \tread the list of packages to be checkoed out from FILE"
-  $ECHO "    --https        \tuse https, rather than ssh to access your personal repository"
-  $ECHO "    --ssh          \tuse ssh, rather than https to access the official repository"
   $ECHO "-q, --quiet, -z    \tdo not print out progress"
-  $ECHO "-y, --yes          \tassume yes to all questions"
-  exit $1
+  if [ "$COMMAND_NAME" == "cms-addpkg" ]; then
+    $ECHO "    --https        \tuse https, rather than ssh to access your personal repository"
+    $ECHO "    --ssh          \tuse ssh, rather than https to access the official repository"
+    $ECHO "-y, --yes          \tassume yes to all questions"
+  fi
+  exit $2
+}
+
+checkPkgFile () {
+  INPUT_FILE="$1"
+  CURRENT_BRANCH="$2"
+  for PKG_NAME in `cat $INPUT_FILE`; do
```

Comment:
```for PKG_NAME in $(< $INPUT_FILE); do```
is negligibly more efficient, as it does not need to spawn a child process (I'm not suggesting to change it, just FYI)